dds/DCPS/XTypes/DynamicType.cpp

```diff
@@ -7,85 +7,120 @@
 #include <dds/DdsDcpsInfrastructureC.h>
 
 OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+DDS::DynamicTypeMembersById_ptr TAO::Objref_Traits<DDS::DynamicTypeMembersById>::nil()
+{
+  // TODO: Help.
```

Comment:
Since we don't have support for `map`s in IDL, I defined DynamicTypeMembersById and DynamicTypeMembersByName as interfaces.

It seems like nil should be easy to implement, but how about `release below`?

Reply:
These types are going to get used extensively when processing a DynamicData so the map feature is useful.

Reply:
See https://github.com/DOCGroup/ACE_TAO/pull/1842 for maps

If they're changed to interfaces the, API for functions that pass/return them will be significantly different.